snmp/tests/test_check.py

```diff
@@ -218,6 +217,26 @@ def test_table(aggregator):
     aggregator.all_metrics_asserted()
 
 
+def test_resolved_table(aggregator):
+    instance = common.generate_instance_config(common.RESOLVED_TABULAR_OBJECTS)
+    check = common.create_check(instance)
+
+    check.check(instance)
+
+    for symbol in common.TABULAR_OBJECTS[0]['symbols']:
```

Comment:
```suggestion
    for symbol in common. RESOLVED_TABULAR_OBJECTS[0]['symbols']:
```

Reply:
No that's on purpose I want to use the original object.